Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -195,5 +195,13 @@ public static IEnumerable<FluentLayout> FullSizeOf(this UIView view, UIView pare
 
 			return layouts;
 		}
+        /// <summary>
+        /// the AspecRatio of view
+        /// </summary>
+        /// <param name="view"></param>
+        /// <param name="scale">width/height</param>
+        /// <returns></returns>
+        public static FluentLayout AspecRatio(this UIView view, nfloat? scale = null)
```

Comment:
Looks like you're missing a `t` in the name here. Also a minor nit, I think a name of `WithAspectRatio` would be a good fit with the other method names in here.